ubi-ruby-builder/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+
+RUBY_MAJOR_VERSION=2.5
+RUBY_FULL_VERSION=2.5.8
+
+docker build -t ubi-ruby-builder:"$RUBY_FULL_VERSION-fips" \
+  --build-arg RUBY_MAJOR_VERSION="$RUBY_MAJOR_VERSION" \
+  --build-arg RUBY_FULL_VERSION="$RUBY_FULL_VERSION" \
```

Comment:
if we add UBI_VERSION and OPENSSL_BUILDER_TAG args, they'd need to be added here too